gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/OrganizationUserServiceImpl.java

```diff
@@ -132,9 +132,11 @@ public Single<User> createGraviteeUser(Organization organization, NewUser newUse
                                     newUser.setClient(null);
                                     // user is flagged as internal user
                                     newUser.setInternal(true);
-                                    String password = newUser.getPassword();
-                                    if (password == null || !passwordService.isValid(password)) {
-                                        return Single.error(InvalidPasswordException.of("Field [password] is invalid", "invalid_password_value"));
+                                    if (!Boolean.TRUE.equals(newUser.getServiceAccount())) {
```

Comment:
```suggestion
                                    if (Boolean.FALSE.equals(newUser.getServiceAccount())) {
```

Reply:
getServiceAccount can be null. I used `if(!Boolean.TRUE.equals(newUser.getServiceAccount()))` instead of doing this: `if(newUser.getServiceAccount() == null || newUser.getServiceAccount().equals(Boolean.FALSE))`